tests/test_filtsmooth/test_particlefiltsmooth/test_particle_filter.py

```diff
@@ -31,44 +31,22 @@ def test_effective_number_of_events():
 
 @pytest.fixture
 def num_particles():
-    return 4
+    return 10
```

Comment:
do we still need 10? What do you think?

Reply:
For me, it fails sometimes, if I use 4. Shall we just use 10 to be sure?

Reply:
Ayt